### DISCUSSION TITLE
Is there a recommended way to manage code snippets?

### DISCUSSION THREAD

**Discussion by liviaerxin** [foambubble/foam#1249] [Ideas]
It's able to:
1. support metadata for tagging explore
2. reside in a separate folder but easy to be linked into other docs

**Comment by riccardoferretti** [comment]
Hey @liviaerxin, there is nothing specific in Foam for managing code snippets. It's not one of my use cases so I have not much to say about it, nor I think I really understand the problem you are trying to solve, maybe share a couple of use cases?
